src/controllers/apiAuth.controller.ts

```diff
@@ -0,0 +1,62 @@
+import passport from "passport";
+import passportCustom from "passport-custom";
+import bcrypt from "bcrypt";
+const CustomStrategy = passportCustom.Strategy;
```

Comment:
Did you try

```typescript
import { Strategy: CustomStrategy } from "passport-custom"
```

and did it work when you do?

If it does work, it should be more neater than this.